regression/scripts/ramjet_network/ramjet_network.py

```diff
@@ -0,0 +1,245 @@
+    
+    conditions.freestream.pressure           = ones_1col*atmo_data.pressure
+    conditions.freestream.temperature        = ones_1col*atmo_data.temperature
+    conditions.freestream.density            = ones_1col*atmo_data.density
+    conditions.freestream.dynamic_viscosity  = ones_1col* atmo_data.dynamic_viscosity
+    conditions.freestream.gravity            = ones_1col*9.81
+    conditions.freestream.isentropic_expansion_factor = working_fluid.compute_gamma(conditions.freestream.temperature,conditions.freestream.pressure)
+    conditions.freestream.Cp                 = working_fluid.compute_cp(conditions.freestream.temperature,conditions.freestream.pressure)                                                                               
+    conditions.freestream.R                  = working_fluid.gas_specific_constant
+    conditions.freestream.speed_of_sound     = ones_1col* atmo_data.speed_of_sound
+    conditions.freestream.velocity           = conditions.M * conditions.freestream.speed_of_sound
+    conditions.velocity                      = conditions.M * conditions.freestream.speed_of_sound
+    conditions.q                             = 0.5*conditions.freestream.density*conditions.velocity**2
+    conditions.g0                            = conditions.freestream.gravity
+    
+    # propulsion conditions
+    conditions.propulsion.throttle           =  ones_1col*1.0
+
+    # ------------------------------------------------------------------
+    #   Design/sizing conditions
+    # ------------------------------------------------------------------    
+    
+    # --- Conditions        
+    ones_1col = np.ones([1,1])    
+    
+    # setup conditions
+    conditions_sizing = SUAVE.Analyses.Mission.Segments.Conditions.Aerodynamics()
+
+    # freestream conditions
+    conditions_sizing.freestream.mach_number        = ones_1col*2.5
+    conditions_sizing.M                             = conditions_sizing.freestream.mach_number
+    conditions_sizing.freestream.altitude           = ones_1col*10000.  
+    
+    atmosphere                                      = SUAVE.Analyses.Atmospheric.US_Standard_1976()
+    atmo_data                                       = atmosphere.compute_values(conditions_sizing.freestream.altitude,0,True) 
+    working_fluid                                   = SUAVE.Attributes.Gases.Air()    
+
+    conditions_sizing.freestream.pressure           = ones_1col*atmo_data.pressure
+    conditions_sizing.freestream.temperature        = ones_1col*atmo_data.temperature
+    conditions_sizing.freestream.density            = ones_1col*atmo_data.density
+    conditions_sizing.freestream.dynamic_viscosity  = ones_1col*atmo_data.dynamic_viscosity
+    conditions_sizing.freestream.gravity            = ones_1col*9.81
+    conditions_sizing.freestream.isentropic_expansion_factor = working_fluid.compute_gamma(conditions_sizing.freestream.temperature,conditions_sizing.freestream.pressure)
```

Comment:
tabulation.

Hint you can do something like:
freestream = conditions_sizing.freestream
freestream.pressure = ...